src/views/block_group_viewer.rs

```diff
@@ -736,24 +803,10 @@ impl<'a> Viewer<'a> {
                             ctx,
                             label::END,
                             (x3, y),
-                            Style::default().fg(Color::DarkGray),
+                            Style::default().fg(col("text_muted").unwrap()),
                         );
                     }
                 }
-
-                // Draw a cursor if no block is currently selected
```

Comment:
Why is this section being deleted?  Is this the case where the splash screen is shown?